Source/Core/DiscIO/DiscScrubber.cpp

```diff
@@ -45,6 +45,13 @@ bool DiscScrubber::SetupScrub(const Volume* disc)
   return success;
 }
 
+std::unique_ptr<DiscScrubber> DiscScrubber::CopyScrubber() const
+{
+  auto retval = std::unique_ptr<DiscScrubber>(new DiscScrubber());
+  retval->SetupScrub(CreateVolume(m_disc->GetBlobReader().CopyReader()).get());
```

Comment:
This is unsafe. The new scrubber keeps a pointer to the volume, but the volume gets destroyed once this line is done executing.

Reply:
I must be misunderstanding how unique_ptr is being used. I'll try to see how other calls to CreateVolume work.

Reply:
This hasn't been resolved.